internal/session/handle_id.go

```diff
@@ -0,0 +1,42 @@
+package session
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ProtonMail/gluon/imap"
+	"github.com/ProtonMail/gluon/internal"
+	"github.com/ProtonMail/gluon/internal/parser/proto"
+	"github.com/ProtonMail/gluon/internal/response"
+)
+
+func prepareServerResponse(info *internal.VersionInfo) *response.IdResponse {
+	return &response.IdResponse{
+		ID: imap.NewIMAPIDFromVersionInfo(info),
+	}
+}
+
+func (s *Session) handleIDGet(ctx context.Context, versionInfo *internal.VersionInfo, tag string, ch chan response.Response) error {
```

Comment:
Why pass in the version info as a parameter when it's already available in the session struct as a data member?

Reply:
:facepalm:

Reply:
Fixed.